public-ui/src/environments/environment.prod.ts

```diff
@@ -8,8 +8,8 @@ export const environment = {
   gtmId: 'GTM-NQ9XDTW',
   gtmAuth: 'ixtwsbMAbR3hx5st_ek2jg',
   gtmPreview: 'env-65',
-  copeFlag: false,
+  copeFlag: true,
   preCopeFlag: true,
```

Comment:
preCopeFlag should be false

Reply:
I just saw that the precope flag is not used anywhere. Cleaned it up.